src/ECharts/Style.elm

```diff
@@ -679,6 +696,124 @@ encodeLegendAlignmentValue value =
 
 
 
+-- xAxis
+
+
+{-| describe the xAxis
+[option](https://ecomfe.github.io/echarts-doc/public/en/option.html#xAxis)
+-}
+type alias XAxisOption =
```

Comment:
You should also include the `@docs` in TOC of the module
the TOC begin from Line 53